client/components/mobile-phone-change/mobile-phone-change.js

```diff
@@ -272,4 +252,6 @@ MobilePhoneChange.propTypes = {
   cookies: PropTypes.instanceOf(Cookies).isRequired,
   logout: PropTypes.func.isRequired,
   verifyMobileNumber: PropTypes.func.isRequired,
+  userData: PropTypes.object.isRequired,
```

Comment:
I think we need to define this further, at least the required attributes we need for the code to work